pkg/plugin/server/http.go

```diff
@@ -40,10 +43,24 @@ type httpPlugin struct {
 }
 
 func NewHTTPPluginOptions(options HTTPPluginOptions) Plugin {
+	var client *http.Client
+	if options.TlsVerify == false {
+		tr := &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		}
+		client = &http.Client{Transport: tr}
+	} else {
+		client = &http.Client{}
+	}
+
+	var url = fmt.Sprintf("%s%s", options.Addr, options.Path)
```

Comment:
how about adding `schema`, e.g.
```
if !options.TlsVerify {
  schema = "http"
} else {
  schema = "https"
}
...
url = fmt.Sprintf("%s://%s%s", schema, addr, path)
```

add I'll appreciate it very much if you could add [e2e test](https://github.com/fatedier/frp/blob/HEAD/test/e2e/plugin/client_plugins.go)

Reply:
`scheme` is not releated to `TLSVerify` since `TLSVerify` is only used for `HTTPS`.

Reply:
Yeah, what fatedier said. TLSVerify just allows us to skip certificate verification on a https connection. You'd probably not want that in a production setting, but it can be useful during development. It certainly is for me at least.

Reply:
@mikehardenize @fatedier try 
```
[common]
bind_port = 7000

[plugin.user-manager]
addr = localhost:9000
path = /handler
ops = Login
tls_verify = false
```
some errer will happen, because the url(http://localhost:9000) not match tls config(skip: true). So I think it better to build the "suitable schema"(not "http" always) by tls_verify when schem not exists.

Reply:
Are you saying that the existence of a `tls_verify` config item should dictate whether or not we use https as the scheme and that we should never accept a scheme in the value for `addr`? I guess it *could* work that way, but why is that way better than the way I already did it?

Reply:
It's nothing about `TLSVerify`.  We can't detect if it's HTTPS server by this.

1. Default schema is `HTTP`.
2. If schema is `HTTPS`, build TLS transport with `TLSVerify`.

Reply:
@mikehardenize maybe what I wrote confuses you. Now I reorganize it:

1. if `addr` is not specified the schema(`http` or `https`), we will use default schema `http`. Then follow 2.
2. if the schema of `addr` is `http`, no matter what `tls_verify ` value is, we will use default transport. That mean `tls_verify` will not decide which transport to use. 
3. if and only if  the schema of `addr` is `https` and `tls_verify` value is `false`, now, we will use transport with tlsconfig(InsecureSkipVerify: true).

what do you think ?

Reply:
Ok. I think I understand now. Although I'm not sure what practical difference it makes? I've just pushed up a commit with that change.

Reply:
![image](https://user-images.githubusercontent.com/18277081/100414932-e7235b80-30b5-11eb-9b4e-d5967c19dadf.png)

I  hope case#4 & case$6 can work. I think it "practical difference".

I would appreciate it if you can rebase commits.

Reply:
I don't know if you're asking me to change anything. Also, I don't know what you want me to rebase against as there are no new commits since mine. I think there might be a language barrier that is preventing this from progressing. Would it make sense for you to make any required changes, rather than trying to get me to make them at this point? Alternatively you could just click approve and then merge and we can both move on?